pkgs/tools/security/tcb/fix-makefiles.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/Make.defs b/Make.defs
```

Comment:
This should be upstreamed.

Reply:
Do I need to replace the patch file?

Reply:
Nope. This is not what I meant.

I am saying that this patch should be sent to the TCB programmer's team.

Reply:
These changes have more to do with NixOS compatibility. There is no way to check on other systems right now.

Reply:
I don't think so.
Many other projects follow GNU convention about installation directories; indeed even CMake has support to it:

https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

https://www.gnu.org/prep/standards/html_node/Directory-Variables.html

Reply:
@solardiz is it possible to add some of the changes in the patch to upstream? Or add build support to NixOS?

Reply:
> @solardiz is it possible to add some of the changes in the patch to upstream? Or add build support to NixOS?

Maybe, I doubt we'd review/discuss/revise them sufficiently for inclusion upstream `tcb` soon. They look questionable to me. Thanks for suggesting this, though! CC: @ldv-alt @besser82